mailet/src/site/xdoc/mailetdocs-maven-plugin/index.xml

```diff
@@ -31,12 +31,16 @@
                 <a href='http://james.apache.org/mailet/standard/mailet-report.html'>report</a>.
             </p>
             <subsection name='Goals Overview'>
-                <p>MailetDocs plugin has one goal:</p>
+                <p>MailetDocs plugin has two goals:</p>
                 <ul>
                     <li>
                         <a href='mailetdocs-mojo.html'>mailetdocs:mailetdocs</a>
                         Generates catalog and reports on mailets and matchers
                     </li>
+                    <li>
+                        <a href='mailetdocs-mojo.html'>mailetdocs:aggregate</a>
+                        Generates catalog and reports on mailets and matchers on a project and all its subprojects
```

Comment:
That's hard to understand the difference with the previous goal. Can you modifiy the previous to tell its limits?